devenv/lib/gcloud.py

```diff
@@ -27,13 +27,17 @@ def _install(url: str, sha256: str, into: str) -> None:
     fs.write_script(
         f"{into}/gcloud",
         f"""#!/bin/sh
-exec /usr/bin/env CLOUDSDK_PYTHON={root}/python/bin/python3 PATH={into}/google-cloud-sdk/bin:$PATH gcloud "$@"
+export CLOUDSDK_PYTHON="{root}/python/bin/python3" \
+       PATH="{into}/google-cloud-sdk/bin:${{PATH}}"
```

Comment:
technically we should be using `shlex.quote` on all of these values (protect against someone `mkdir '$(cat /etc/passwd)'`)

Reply:
should i make write_script better by passing an optional format dict that'll then shell escape all those tokens?

Reply:
if it makes it simpler sure that sounds decent